src/ORM/FieldType/DBClassName.php

```diff
@@ -95,44 +85,37 @@ public function getBaseClass()
     /**
      * Get the base name of the current class
      * Useful as a non-fully qualified CSS Class name in templates.
-     *
-     * @return string|null
      */
-    public function getShortName()
+    public function getShortName(): string
     {
         $value = $this->getValue();
         if (empty($value) || !ClassInfo::exists($value)) {
-            return null;
+            return '';
         }
         return ClassInfo::shortName($value);
     }
 
     /**
      * Assign the base class
-     *
-     * @param string $baseClass
-     * @return $this
      */
-    public function setBaseClass($baseClass)
+    public function setBaseClass(?string $baseClass): static
```

Comment:
Nullable param to allow resetting to the default value since the property itself is nullable